src/main/resources/apikey.properties

```diff
@@ -1,43 +1,55 @@
-spring.datasource.url                       = [REMOVED]
-spring.datasource.username                  = [REMOVED]
-spring.datasource.password                  = [REMOVED]
-
-spring.datasource.hikari.connectionTimeout  = 30000
-spring.datasource.hikari.idleTimeout        = 600000
-spring.datasource.hikari.maxLifetime        = 1800000
-spring.datasource.hikari.minimumIdle        = 1
-spring.datasource.hikari.maximumPoolSize    = 20
-
-spring.mail.host     = [REMOVED]
-spring.mail.username = [REMOVED]
-spring.mail.password = [REMOVED]
-spring.mail.port     = 0
-
-europeanaMail.sentFrom = [REMOVED]
-europeanaMail.copyTo   = [REMOVED]
-
-keycloak.realm                   = [REMOVED]
-keycloak.realmPublicKey          = [REMOVED]
-keycloak.authserverurl           = [REMOVED]
-keycloak.useResourceRoleMappings = true
-keycloak.managerClientId         = [REMOVED]
-keycloak.managerClientSecret     = [REMOVED]
-
-recaptcha.secret                  = [REMOVED]
-recaptcha.verification-url.scheme = https
-recaptcha.verification-url.host   = www.google.com
-recaptcha.verification-url.path   = /recaptcha/api/siteverify
+#server.port                                = 8443
+#server.ssl.enabled                         = true
+#server.ssl.key-store-type                  = PKCS12
+#server.ssl.key-store                       = classpath:keystore/apikey.p12
+#server.ssl.key-store-password              = enigm@Torant1
+#server.ssl.key-alias                       = apikey
+
+spring.jpa.hibernate.ddl-auto              = validate
+spring.jpa.database                        = POSTGRESQL
+spring.jpa.show-sql                        = true
+
+spring.datasource.driverClassName          = org.postgresql.Driver
+spring.datasource.url                      = [REMOVED]
+spring.datasource.username                 = [REMOVED]
+spring.datasource.password                 = [REMOVED]
+
+spring.datasource.hikari.connectionTimeout = 30000
+spring.datasource.hikari.idleTimeout       = 600000
+spring.datasource.hikari.maxLifetime       = 1800000
+spring.datasource.hikari.minimumIdle       = 1
+spring.datasource.hikari.maximumPoolSize   = 20
+
+spring.mail.host                           = [REMOVED]
+spring.mail.username                       = [REMOVED]
+spring.mail.password                       = [REMOVED]
+spring.mail.port                           = 0
+
+europeana.mail.from                        = noreply@europeana.eu
+europeana.mail.bcc                         = noreply@europeana.eu
+
+# for delete user functionality
+keycloak.user.admin.granttype              = [REMOVED]
+keycloak.user.admin.password               = [REMOVED]
+keycloak.user.admin.username               = [REMOVED]
+keycloak.user.admin.clientid               = [REMOVED]
+keycloak.user.slack.webhook                = [REMOVED]
+keycloak.user.slack.email                  = [REMOVED]
+keycloak.masterPublicKey                   = [REMOVED]
+keycloak.set.api.url                       = https://set-api-test.eanadev.org/set/
```

Comment:
I would be more comfortable not putting a link to a test server in our configuration.